lib/permissions.js

```diff
@@ -37,6 +33,6 @@ Permissions.prototype.authorizeAPI = function (req, res, cb) {
 };
 
 Permissions.prototype.authorize = function (req, res, cb) {
-  if (req.headers.authorization !== ':') return this.authorizeAPI(req, res, cb);
+  if (req.headers.authorization) return this.authorizeAPI(req, cb);
```

Comment:
This change was failing because we didn't update the `flatsheet-api-client` dependency version. I updated it to 2.0, which was committed, but that lead to other problems, described under the `package.json` diff.